.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,24 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --show-source --statistics
```

Comment:
I'd like to see an actual lint error generated by flake8`. 

Could you deliberately modify some Python code, such that it is dramatically correct but not up to the linting standard? After we get the results from CI confirming the linter is working, revert the commit.

Reply:
`--show-source` option is doing that.

Reply:
pre-commit.ci can autofix errors but GitHub Actions makes that difficult to do.